Fix issue #1260

Phyks opened the pull request:
Fix coding guidelines incoherences. Code should match PSR1/2 now, and
php-cs is set to check it on each commit.

Closes #1260.

Phyks commented:
Still doing some work, I missed something.

Phyks commented:
Ready to be merged / discussed now, sorry.

I added a fix for the Git hook, which was trying to test deleted files.

Note: I added a second commit to fix for PSR0 compatibility, errors being due to the `UPnP` case which did not agree with the PSR0 files naming convention (mixed cases in class name and lowercase in filenames). I am not really sure how relevant this is, and I have no problems discarding this commit if you think it is not a good thing.

Phyks commented:
Closing as the codebase as changed too much and this could just break anything. I think we should freeze PRs and merge such a PR first.

I can resubmit an updated version whenever you want.

Afterster commented:
Yes this was the reason I didn't merged it lately. You can resubmit the PR, it will be the first thing I will merge. Thank you.

Phyks commented:
Ok, thanks. I will try to have a look at it by the beginning of next week.

Phyks commented:
Just a note to remember to change https://github.com/ampache/ampache/pull/1268/files#diff-e461e5544b65b1ab54364194ecb67585R6 everywhere.

Phyks commented:
Also change end of line to `PHP_EOL`. See https://github.com/ampache/ampache/issues/1305#issuecomment-233535426.

Phyks commented:
Reopen with waiting flag to remember to handle it.

Phyks commented:
Ok, I rebased it on `develop` and it should be ready for merge. I hope this did not introduce any side bugs, but I did not see with a rapid test.

I kept 4 separate commits for easier reviewing:
* First commit (Fix issue #1260) is a fix of PSR 1 and 2 deviations, mostly style fixes. I have reviewed them when applying them and it should be ok.
* Second commit (Fix for PSR0 compatibility) is an extra style fix to achieve PSR0 compliance. It involves renaming of some classes, so I prefered to handle it separately. Renamed classes should have been renamed across all files if I did not do any mistake.
* Third commit is the copyright date extension across all files in Ampache project.
* Last commit groups fixes for deviations that were introduced after the rebase.

Phyks commented:
Still, something bothers me. I don't get why Travis build fails on this branch.

Output on my laptop is:
```bash
22:20 phyks@Phyks-laptop /srv/http/ampache % sh ./scripts/tests/syntax.sh                                                                                                                                            ±[issue_1260]
Checking syntax error
Syntax is OK
22:21 phyks@Phyks-laptop /srv/http/ampache % sh ./scripts/tests/codestyle.sh                                                                                                                                         ±[issue_1260]
Checking mandatory formatting/coding standards
Loaded config from ".php_cs"
.................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................
Legend: ?-unknown, I-invalid file syntax, file ignored, .-no changes, F-fixed, E-error
Checked all files in 22.393 seconds, 16.000 MB memory used
Formatting is OK
```

Afterster commented on modules/localplay/upnp/upnpplayer.class.php:
```diff
@@ -69,8 +69,8 @@ private function Playlist()
-        require_once AmpConfig::get('prefix') . '/modules/localplay/upnp/upnpdevice.php';
-        require_once AmpConfig::get('prefix') . '/modules/localplay/upnp/upnpplaylist.php';
+        require_once AmpConfig::get('prefix') . '/modules/localplay/upnp/UPnPDevice.php';
+        require_once AmpConfig::get('prefix') . '/modules/localplay/upnp/UPnPPlaylist.php';
```
Why did you put some letter uppercase here? It is lowercase on the repository and could make linux os to fail.

Phyks replied:
I had a dilemma here. PSR-0 requires the filename to match the one of the defined class name. So we should:
* Either deviate from PSR-0 here
* Rename `UPnPDevice` class to `upnpdevice`, which I don't like much personnally
* Rename the file names

Not sure which is the best.

Afterster replied:
OK I see, then I agree on the changes, thanks.
